modules/javafx.graphics/src/main/java/javafx/concurrent/ScheduledService.java

```diff
@@ -127,16 +127,16 @@
  * <p>For this purposes of this class, any Duration that answers true to {@link javafx.util.Duration#isUnknown()}
  * will treat that duration as if it were Duration.ZERO. Likewise, any Duration which answers true
  * to {@link javafx.util.Duration#isIndefinite()} will be treated as if it were a duration of Double.MAX_VALUE
- * milliseconds. Any null Duration is treated as Duration.ZERO. Any custom implementation of an backoff strategy
+ * milliseconds. Any {@code null} Duration is treated as Duration.ZERO. Any custom implementation of a backoff strategy
```

Comment:
Since you changed `null` to use code style, maybe also do it for `Duration.ZERO`?

Reply:
There are many places that are missing it in this class. Should I do them all while here?